### PR TITLE
Comp alerts

### PR DESCRIPTION
**Pull request by jeetsdev**
Added alerts component.

### CODE REVIEW

File: components/alerts.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+.alert-pri{
+    background-color: var(--text-pri);
+    color: var(--color-white);
+}
+
+.alert-suc{
+    background-color: var(--color-off-green);
+    color: var(--color-green);
+}
+.alert-war{
```
Comment (junaidshaikh-js) [inline components/alerts.css]: Consistent line spacing between your style declaration blocks should be used, Prettier  (VS code extension ) might help with this.

File: css/main.css [hunk 2]
```diff
@@ -17,16 +17,23 @@
...
+    --color-green:hsl(120, 100%, 25%);
+    --color-off-green:hsl(151, 41%, 79%);
+    --color-yellow:hsl(35, 100%, 50%);
+    --color-yellow:hsl(60, 100%, 25%);
+    --color-off-yellow:hsl(60, 41%, 79%);
+    --color-red:hsl(0, 100%, 25%);
+    --color-off-red:hsl(0, 41%, 79%);
 }
 
 html{
```
Comment (junaidshaikh-js) [inline css/main.css]: Can be moved to reset.css if you have one.

File: css/main.css [hunk 3]
```diff
@@ -17,16 +17,23 @@
...
     scroll-behavior: smooth;
 }
 
 body {
```
Comment (junaidshaikh-js) [inline css/main.css:33]: This can also be moved to reset.css

File: documentation/docs.html [hunk 4]
```diff
@@ -45,8 +45,8 @@ <h2 class="headline-3 text-sub-head">Components</h2>
                         <span>
                             <h1 class="headline-1 text-head">Avatars</h1>
```
Comment (junaidshaikh-js) [inline documentation/docs.html]: h1 should not be nested inside span

**Review by znjs — APPROVED** [review]
Looks Good 👍

File: documentation/docs.html [hunk 5]
```diff
@@ -91,6 +92,56 @@ <h2 class="headline-2 text-sub-head">Text avatars</h2>
                         </div>
                     </div>
                 </section>
+                <section id="alerts-sec">
+                    <div class="comp-intro">
+                        <span>
+                            <h1 class="headline-1 text-head">Alerts</h1>
```
Comment (junaidshaikh-js) [inline documentation/docs.html]: h1 should not be nested inside span

File: components/alerts.css (hunk 1 above)
Comment (jeetsdev) [reply components/alerts.css]: Done

File: css/main.css (hunk 2 above)
Comment (jeetsdev) [reply css/main.css]: Done

File: css/main.css (hunk 3 above)
Comment (jeetsdev) [reply css/main.css:33]: > This can also be moved to reset.css

Moved into reset CSS, Thanks.

File: documentation/docs.html (hunk 5 above)
Comment (jeetsdev) [reply documentation/docs.html]: > h1 should not be nested inside span

Done

File: css/main.css [hunk 6]
```diff
@@ -167,28 +172,29 @@ span {
 
 /* * Margin styles here */
 
-.mar-8p{
+.marg-8p {
```
Comment (dikshadhote) [inline css/main.css]: You can form separate file utiity.css for general margin and padding styles

**Review by ayush65 — APPROVED** [review]

**Review by shraddha15-creator — APPROVED** [review]

File: documentation/docs.html [hunk 7]
```diff
@@ -91,6 +90,55 @@ <h2 class="headline-2 text-sub-head">Text avatars</h2>
...
+                        <p class="text-mid">
+                            You can use <span>alerts</span> class to create an alert. Use <span>alert-pri, alert-suc,
+                                alert-war, alert-dan</span> for diffrent types of alerts.
+                            You can use diffrent style classes for diffrent stylling.
+                        </p>
+                        <div class="comp-examples center-flex flex-d-col">
+                            <div class="alerts alert-pri padd-1r marg-1r bor-small text-mid">
+                                <p class="alert">
+                                    This is a Primary alert.
+                                </p>
```
Comment (JowelTisso) [inline documentation/docs.html]: Where is the class "alert" that you have used in the p tag, or is it redundant.

Comment (jeetsdev) [reply documentation/docs.html]: Ohh, It's just redundant. I forgot to remove it.

File: css/main.css (hunk 6 above)
Comment (jeetsdev) [reply css/main.css]: Created a separate file for utility classes.

File: documentation/docs.html (hunk 7 above)
Comment (JowelTisso) [reply documentation/docs.html]: Ok, rest is fine.

**Review by JowelTisso — APPROVED** [review]

**Comment by jeetsdev** [comment]
Added alert components.

**Pull request merged**